core/ElasticSearchBulkGenerator.py

```diff
@@ -329,6 +348,9 @@ def generate_elasticsearch_files(ontology_dir,
         crit_set_dir = f'{ontology_dir}/criteria-sets'
         ElasticSearchGenerator.__build_crit_set_map(context_termcode_hash_to_crit_set, crit_set_dir, namespace_uuid_str)
 
+        terminology_resolver = TerminologyDesignationResolver()
+        terminology_resolver.load_designations("example/code_systems_translations")
```

Comment:
path to folder should no be hardcoded and instead passed from the generate elastic search script